fullscreen.bs

```diff
@@ -496,11 +499,15 @@ following characteristics:
  <code>right</code>, and <code>top</code> is zero.
 </ul>
 
-<p>To <dfn export for="top layer">add</dfn> an <var>element</var> to a <var>top layer</var>, add, or
-move if already present, <var>element</var> on top of <var>top layer</var>.
+<p>To <dfn export for="top layer">add</dfn> an <var>element</var> to a <var>top layer</var>,
+<a for=set>remove</a> it from <var>top layer</var> and then <a for=set>append</a> it to
+<var>top layer</var>.
+
+<p class=note>In other words, <var>element</var> is moved to the end of <var>top layer</var> if it
+is already present.
 
 <p>To <dfn export for="top layer">remove</dfn> an <var>element</var> from a <var>top layer</var>,
-remove <var>element</var> from <var>top layer</var>.
+<a for=set>remove</a> <var>element</var> from <var>top layer</var>.
```

Comment:
Given that these are identical, perhaps we should no longer have a dedicated remove entry point?